ecosystem/sep-0024.md

```diff
@@ -322,22 +402,24 @@ The fields below should be placed in the request body using the `multipart/form-
+| ----------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `asset_code`                  | string                                  | The code of the stellar asset the user wants to receive for their deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object. When `quote_id` is specified, `asset_code` must match the quote's `buy_asset` asset code.                                                                                                                                                                    |
+| `asset_issuer`                | string                                  | (optional) The issuer of the stellar asset the user wants to receive for their deposit with the anchor. If asset_issuer is not provided, the anchor should use the asset issued by themselves as described in their TOML file. When `quote_id` is specified, `asset_issuer` must match the quote's `buy_asset` asset issuer.                                                                                                                                 |
+| `sending_asset`               | string in [Asset Identification Format] | (optional) The asset user wants to send. Note, that this is the asset user initially holds (off-chain or fiat asset).If this is not provided, it will be collected in the interactive flow. When `quote_id` is specified, this parameter is mandatory and must match the quote's `sell_asset` asset code.                                                                                                                                                    |
+| `quote_id`                    | string                                  | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor must respect the conversion rate agreed in that quote. When `quote_id` is set, `asset_code`, `sending_asset` and `amount` are mandatory. In case of a conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`. |
```

Comment:
@Ifropc thanks for the description update.

Is `amount` really mandatory here?
On the description of the `amount` field it says it should be omitted if a `quote_id` was used so both descriptions are conflicting now.

Reply:
No, I got everything mixed up. Will fix description.
amount, assets fields are not required, but if they are set, they must match the quote.

Reply:
Looks good, thanks for fixing those!

I left another comment related to the `quote_id` description (looks like only withdrawal as updated)